docs/competitive-programming/prefix-sum-algorithm.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Prefix Sum Algorithm
+slug: /competitive-programming/prefix-sum-algorithm
+---
+
+## Introduction
+
+In this article, we will learn about Prefix Sum Algorithm.
+Let us say, You are given an array of `N` elements. It's Prefix Sum is another array of same size.
+where PrefixSum[i] is sum of all prefix elements of the given array. Formally, 
```

Comment:
Similarly here. The idea is that the variables, function names, operators etc should be in backticks.